src/view/send/SendByAddress.svelte

```diff
@@ -34,14 +39,24 @@
     }
 
     const send = async () => {
-        if(toAddress && sendAmount) {
+        if (toAddress && sendAmount) {
             await sendNano(account, toAddress, nanoToRaw(sendAmount), balance)
         }
     }
 
+    const scannedAddress = (address: NanoAddress) => {
+        toAddress = address;
+        showCamera = false;
+        sendType = 'address'
+    }
+
 </script>
 
-<LabelledInput languageId="send-address" type="text" on:change={setAddress}/>
-<LabelledInput languageId="send-amount" type="text" on:change={setAmount} bind:value={balanceValue}/>
-<Button languageId="send-max-button" on:click={setMax}/>
-<Button languageId="send-button" on:click={send}/>
+{#if sendType === 'qr' && showCamera}
```

Comment:
This makes us go straight into "qr scanning mode" when selecting QR button. And once we're finished scanning, `showCamera` will be set to false, and we continue in "address"-mode, but with the address set.